specs/Media.md

```diff
@@ -0,0 +1,146 @@
+            MessageBox(nullptr, isMuted ? L"Yes" : L"No", L"Is Muted", MB_OK);
+            return true;
+        case IDM_IS_CURRENTLY_AUDIBLE:
+            BOOL isAudible;
+            m_webView->get_IsCurrentlyAudible(&isAudible);
+            MessageBox(nullptr, isAudible ? L"Yes" : L"No", L"Is Audible", MB_OK);
+            return true;
+    }
+}
+```
+
+## .NET and WinRT
+
+```c#
+    void MuteCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        webView.CoreWebView2.Mute();
+    }
+
+    void UnmuteCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        webView.CoreWebView2.Unmute();
+    }
+
+    void IsMutedCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        bool is_muted = webView.CoreWebView2.IsMuted;
+        MessageBox.Show(this, is_muted? "Is muted": "Is Not muted", "Is Muted or Not");
+    }
+
+    void IsCurrentlyAudibleCmdExecuted(object target, ExecutedRoutedEventArgs e)
+    {
+        bool is_currently_audible = webView.CoreWebView2.IsCurrentlyAudible;
+        MessageBox.Show(this, is_currently_audible ? "Is audible" : "Is not audible", "Is Currently Audible or Not");
+    }
+```
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+
+```IDL
+interface ICoreWebView2StagingIsMutedChangedEventHandler;
+interface ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler;
+
+[uuid(76eceacb-0462-4d94-ac83-423a6793775e), object, pointer_default(unique)]
+interface ICoreWebView2_2 : ICoreWebView2 {
+  /// Adds an event handler for the `IsMutedChanged` event.
+  /// `IsMutedChanged` runs when the mute state changes. The event may run 
+  /// when `Mute` and `Unmute` are called.
+  HRESULT add_IsMutedChanged(
+      [in] ICoreWebView2StagingIsMutedChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with `add_IsMutedChanged`.
+  HRESULT remove_IsMutedChanged(
+      [in] EventRegistrationToken token);
+
+  /// Mutes all audio output from this CoreWebView2.
+  HRESULT Mute();
+  /// Unmutes all audio output from this CoreWebView2.
+  HRESULT Unmute();
+  /// Indicates whether all audio output from this CoreWebView2 is muted or not.
+  [propget] HRESULT IsMuted([out, retval] BOOL* isMuted);
+  
+  /// Adds an event handler for the `IsCurrentlyAudibleChanged` event.
+  /// `IsCurrentlyAudibleChanged` runs when the audible state changes.
+  HRESULT add_IsCurrentlyAudibleChanged(
+      [in] ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Remove an event handler previously added with `add_IsCurrentlyAudibleChanged`.
+  HRESULT remove_IsCurrentlyAudibleChanged(
+      [in] EventRegistrationToken token);
+  
+  /// Indicates whether any audio output from this CoreWebView2 is audible.
+  /// CoreWebView2 can be audible when muted, IsCurrentlyAudible is used to indicate
```

Comment:
I assume when you say 'can be audible when muted' you mean IsCurrentlyAudible can be true even when IsMuted is also true. Maybe a more explicit way to phrase this would be like the following:

```suggestion
  /// IsCurrentlyAudible is used to indicate if there is audio or media currently playing and can be true even if IsMuted is true or the volume is turned off.
```